vcloud.yaml

```diff
@@ -1,11 +1,334 @@
+        405:
+          $ref: "#/components/responses/code-405"
+        422:
+          $ref: "#/components/responses/code-422"
+        500:
+          $ref: "#/components/responses/code-500"
+        503:
+          $ref: "#/components/responses/code-503"
+        504:
+          $ref: "#/components/responses/code-504"
+  /status:
     get:
-      summary: Placeholder
+      tags:
+        - Status Request
+      summary: Check V-Cloud availability
+      description:
+        The response indicates the status and availability of V-Cloud.
+        Contact support@vocitec.com if you are experiencing an operational issue.
       responses:
         200:
-          description: Success
+          description: V-Cloud is available. You do not need to pass a token to make this call.
+          content:
+            text/plain:
+              example: OK
+components:
+  schemas:
+    #-------------------------------
+    # Request body form-data
+    #-------------------------------
+    transcription-parameters:
+      type: object
+      required:
+        - file
+        - token
+      properties:
+        file:
+          type: string
+          description: Specify a single `.wav` audio file or a zip archive
+            containing multiple `.wav` audio files to transcribe. Refer to
+            the `transcode` parameter to use audio formats other than `.wav`.
+          format: binary
+        token:
+          type: string
+          description: Unique string used to authenticate and authorize the
+            request.
+          example: "123e4567e89b12d3a456426655440000"
+        url:
+          type: string
+          description: Use as an alternative to the `file` parameter for sending
+            audio in a request. The provided URL must support HTTP `GET` and
+            return a `Content-Length` header when queried.
+          example: "https://username:password@hostname.com/path/to/sample.wav"
+        callback:
+          type: string
+          description: Provide a callback address for V-Cloud to send the
+            results when the transcription process is complete.
+          example: "https://username:password@hostname.com"
+        diarize:
+          type: boolean
+          description: Use `diarize` to identify distinct speakers on mono
+            (single channel) audio and segment detected speech into separate
+            channels, which are identified in JSON output.
+          default: false
+        emotion:
+          type: boolean
+          description: Emotion detection feature that uses a synthesis of
+            acoustic features and word sentiment scores to determine if a
+            given utterance is Mostly Positive, Positive, Neutral, Negative,
+            or Mostly Negative.
+          default: false
+        gender:
+          type: boolean
+          description: Gender detection feature that attempts to determine
+            if the speaker is male or female in each utterance.
```

Comment:
Need to address the `male`/`female` thing here.  Can we put it into the context of voice deepness / hertz / etc.?